clustering_experiments.ipynb

```diff
@@ -52,54 +55,84 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "n_attrs=500\n",
+    "n_attrs=20\n",
     "max_iter=1000\n",
-    "n_clusters=20"
+    "n_clusters=25"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {},
```

Comment:
for t-SNE plot - which are the opaque items?

Reply:
that title is old, it this only shows users